src/hyperion/experiment_plans/flyscan_xray_centre_plan.py

```diff
@@ -108,7 +110,7 @@ def set_aperture_for_bbox_size(
     else:
         aperture_size_positions = aperture_device.aperture_positions.LARGE
         selected_aperture = "LARGE_APERTURE"
-    hyperion.log.LOGGER.info(
+    LOGGER.info(
```

Comment:
Yay! I'm glad we standardised this. If I'm being picky I would have liked to see it in a different PR but it's fine.